diff --git a/node_modules/react-native-safe-area-context/android/build.gradle b/node_modules/react-native-safe-area-context/android/build.gradle
index f324b9a..c1258ab 100644
--- a/node_modules/react-native-safe-area-context/android/build.gradle
+++ b/node_modules/react-native-safe-area-context/android/build.gradle
@@ -73,6 +73,9 @@ android {
     lintOptions{
         abortOnError false
     }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
 
     packagingOptions {
         // For some reason gradle only complains about the duplicated version of libreact_render libraries
