diff --git a/node_modules/react-native-safe-area-context/android/build.gradle b/node_modules/react-native-safe-area-context/android/build.gradle
index f324b9a..56e46ae 100644
--- a/node_modules/react-native-safe-area-context/android/build.gradle
+++ b/node_modules/react-native-safe-area-context/android/build.gradle
@@ -73,6 +73,9 @@ android {
     lintOptions{
         abortOnError false
     }
+    kotlinOptions {
+        jvmTarget = '17'
+    }
 
     packagingOptions {
         // For some reason gradle only complains about the duplicated version of libreact_render libraries
@@ -86,7 +89,7 @@ android {
             if (isNewArchitectureEnabled()) {
                 srcDirs += [
                     "src/fabric/java",
-                    "${project.buildDir}/generated/source/codegen/java"
+                    "${project.layout.buildDirectory.get().asFile}/generated/source/codegen/java"
                 ]
             } else {
                 srcDirs += [
