diff --git a/node_modules/react-native-purchases/android/build.gradle b/node_modules/react-native-purchases/android/build.gradle
index 5cee90c..0589948 100644
--- a/node_modules/react-native-purchases/android/build.gradle
+++ b/node_modules/react-native-purchases/android/build.gradle
@@ -25,6 +25,8 @@ def getExtOrIntegerDefault(name) {
 android {
     compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
 
+    namespace "com.revenuecat.purchases.react"
+
     defaultConfig {
         minSdkVersion getExtOrIntegerDefault('minSdkVersion')
         targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
@@ -44,6 +46,9 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
 }
 
 repositories {
